grpc/binlog/v1alpha/binarylog.proto

```diff
@@ -35,7 +35,31 @@ message GrpcLogEntry {
     RECV_INITIAL_METADATA = 4;
     RECV_TRAILING_METADATA = 5;
     RECV_MESSAGE = 6;
-  };
+  }
+
+  // Enumerates the type of logs
+  enum EventType {
+    UNKNOWN_EVENT_TYPE = 0;
+    // Headers sent from client to server
+    REQUEST_HEADERS = 1;
```

Comment:
This is related to my comments in grpc/grpc#15460: can we standardize on "header" and "trailer" instead of "headers" and "trailers"?  This avoids the awkwardness of things like "the client sends a headers"

Reply:
Ack, I think this would be an improvement